types/database/schema/ChannelList.types.ts

```diff
@@ -1,4 +1,6 @@
 /* eslint-disable no-shadow */
+import {SQLiteDatabase} from 'react-native-sqlite-storage';
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `SQLiteDatabase` from 'react-native-sqlite-storage' has been added. Ensure that this package is installed and the correct version is specified in your `package.json`. If it's not used, consider removing it to avoid unnecessary imports.

```diff
+ import {SQLiteDatabase} from 'react-native-sqlite-storage';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->